pkg/app/master/commands/clifvgetter.go

```diff
@@ -407,3 +408,30 @@ func GetDockerClientConfig(ctx *cli.Context) *config.DockerClient {
 
 	return config
 }
+
+func validateAndCleanEnvVariables(envList []string, xc *app.ExecutionContext) []string {
+	xc.Out.Info("Validating and cleaning environment variables")
+
+	envStaging := envList[:0]
+	for i, env := range envList {
+		envKeyValue := strings.Split(env, "=")
```

Comment:
speaking of `ParseEnvFile` and its basic validation. your validation logic should start with what it has before trying to do any kind of key/value component length checks.